tools/accuracy_checker/configs/vehicle-license-plate-detection-barrier-0123.yml

```diff
@@ -0,0 +1,55 @@
+models:
+  - name: vehicle-license-plate-detection-barrier-0123
+
+    launchers:
+      - framework: tf
+        device: CPU
+        model:   object_detection/barrier/tf/0123/model/model.pb.frozen
+        adapter: ssd
+
+      - framework: dlsdk
+        tags:
+          - FP32
+        model: public/vehicle-license-plate-detection-barrier-0123/FP32/vehicle-license-plate-detection-barrier-0123.xml
+        weights: public/vehicle-license-plate-detection-barrier-0123/FP32/vehicle-license-plate-detection-barrier-0123.bin
+        adapter: ssd
+
+      - framework: dlsdk
+        tags:
+          - FP16
+        model: public/vehicle-license-plate-detection-barrier-0123/FP16/vehicle-license-plate-detection-barrier-0123.xml
+        weights: public/vehicle-license-plate-detection-barrier-0123/FP16/vehicle-license-plate-detection-barrier-0123.bin
+        adapter: ssd
+
+    datasets:
+      - name: bitvehicle_dataset
+        data_source: BITVehicle/annotation_val/data
+        annotation: bitvehicle.pickle
+        dataset_meta: bitvehicle.json
```

Comment:
could you please change this section on 
```yml
- name: vehicle_license_plate_detection
```
like in config for [vehicle-license-plate-detection-barrier-0106](https://github.com/opencv/open_model_zoo/blob/master/tools/accuracy_checker/configs/vehicle-license-plate-detection-barrier-0106.yml#L30)

Reply:
@eaidova Should I delete data_source, annotation and dataset_meta fields in this section?

Reply:
yes, all right, and change name to `vehicle_license_plate_detection`

Reply:
it needs for configs reusage in light-weight accuracy validation (on 10 images)

Reply:
Thanks!